airflow/jobs/scheduler_job.py

```diff
@@ -686,10 +686,6 @@ def _process_dags(self, dagbag, dags, tis_out):
         """
         check_slas = conf.getboolean('core', 'CHECK_SLAS', fallback=True)
         for dag in dags:
-            dag = dagbag.get_dag(dag.dag_id)
```

Comment:
This will never happen because we have loaded this file one level up.

Reply:
If that is the case, do we even need dagbag in the method signature?

Reply:
No. I removed in fixup, but I forgot to push changes.